x-pack/plugins/security_solution/public/management/components/privileged_route/privileged_route.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { ComponentType } from 'react';
+import React, { memo } from 'react';
+import { Route } from '@kbn/kibana-react-plugin/public';
+import { NoPrivilegesPage } from '../../../common/components/no_privileges';
+import { useIsExperimentalFeatureEnabled } from '../../../common/hooks/use_experimental_features';
+import { NoPermissions } from '../no_permissons';
+import { AdministrationSubTab } from '../../types';
+
+export interface PrivilegedRouteProps {
+  path: string;
+  component: ComponentType<{}>;
+  privilege: boolean;
```

Comment:
Suggestion on the naming of this prop: maybe rename it to `hasPrivilege`?

`privilege` (to me) makes it sound like you pass along the privilege this page needs (ex. `canRead**`). Since its just a boolean, `hasPrivilege` might be better?

Reply:
You are totally right, thanks for the suggestion!
39400ab